simulations/overview.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Overview
+---
+
+import { Steps } from "nextra/components";
+import { Option } from "/components/Option";
+
+The Simulations visualizer in LangWatch provides a powerful way to inspect and analyze the results of your agent tests built with the [`scenario`](https://github.com/langwatch/scenario) library. It offers a user-friendly interface to dig into simulation runs, helping you debug your agent's behavior and collaborate with your team.
```

Comment:
a lot of people will likely arrive here having no clue what simulations are, before we talk about visualizing them

maybe we could add a paragraph before that this is really about Agent Testing, point them that scenario exist, that this is the best way to test agents etc?

you can use those as inspirations, maybe:

https://scenario.langwatch.ai/
https://scenario.langwatch.ai/introduction/simulation-based-testing

and *then* we get into that you can visualize it in langwatch, but first I want to push people that arrive here to understand what this concept even is, even if it's a bit redundant with Scenario docs itself

wdyt?